plugins/modules/inventory_intent.py

```diff
@@ -2886,10 +2965,10 @@ def verify_diff_deleted(self, config):
 
         if device_delete_flag:
             self.status = "success"
-            self.msg = "Requested Devices - {0} Deleted from Cisco DNA Center and Deletion verified.".format(str(input_devices))
-            self.log(self.msg)
+            self.msg = "Requested Devices '{0}' Deleted from Cisco Catalyst Center and Deletion verified.".format(str(input_devices))
```

Comment:
self.msg = "Requested devices '{0}' have been successfully deleted from Cisco Catalyst Center and their deletion has been verified.".format(str(input_devices))

Reply:
added.